staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/validation/validation.go

```diff
@@ -62,6 +64,14 @@ const (
 	StaticEstimatedCRDCostLimit = 100000000
 )
 
+var supportedValidationReason = sets.NewString(
+	string(field.ErrorTypeInvalid),
+	string(field.ErrorTypeForbidden),
+	string(field.ErrorTypeNotFound),
+	string(field.ErrorTypeTooLong),
+	string(field.ErrorTypeRequired),
+)
```

Comment:
I did a quick sweep of the field errors, and `ErrorTypeDuplicate` was the only one not in this list that seemed likely to be emitted from CEL validation and not caught by openapi validation.

The others I considered were `ErrorTypeNotSupported` (more likely to be generated by an `enum` validation failure) and `ErrorTypeTooMany` (more likely to be generated by exceeding `maxItems`)

Reply:
I have added those three:)